src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -40,6 +40,18 @@ public CommandResult execute() {
         return new CommandResult(getMessageForPersonListShownSummary(personsFound), personsFound);
     }
 
+    /**
+     * Changes String in Set<String> to lower case.
+     * @param keywords
+     * @param setToChange
+     */
+    private void changeToLowercaseKeywords(Set<String> keywords, HashSet<String> setToChange){
```

Comment:
Variable names must be in camelCase. (if you are using the "lower case" spelling instead of "lowercase", standardize the casing in other variable/method names as well. Refer to point 3 of https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Java.adoc#naming-conventions